botCommands/simpleCommands.test.js

```diff
@@ -77,7 +77,7 @@ describe('/smart', () => {
       ['/smrt'],
       ['/smarts'],
       ['```function("/smart", () => {}```'],
-      // ['/smart'],
+      //! ['/smart'],
```

Comment:
['/smart'] is mentioned both in pass test and fail tests. 😕

Reply:
all the other 5 comments that I marked, are similar in the sense that they are both written in pass cases and fail cases.